[ADD] Manufacturing: Production analysis report

hojo-odoo opened the pull request:
This is a re-upload of this [PR](https://github.com/odoo/documentation/pull/10133), which was messed up during a squash. All of the comments from the first round of review by @Felicious have already been implemented, so I'm pushing this directly to final review.

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/10155.png)](https://mergebot.odoo.com/odoo/documentation/pull/10155)

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+represents a specific statistic about |MOs| in the database. Choose a measure by clicking the
+:guilabel:`Measures` :icon:`fa-caret-down` button, and selecting one of the following options from
+the drop-down menu:
+
```
I'm also noticing that some of these (in the list below) aren't showing up on the runbot I am using for this review... so, I'd strongly suggest a double-check on this

hojo-odoo replied:
I've added a mention about how the options change based on different settings.

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+- :guilabel:`Quantity Produced`: the total number of units of the product that have actually been
+  produced.
+- :guilabel:`Total Component Cost`: the total amount spent on the product's components, across every
+  MO for the product.
```
```suggestion
  |MO| for the product.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+Views
+=====
+
+By default, the production analysis dashboard displays data using the *line chart* graph type of the
```
```suggestion
By default, the :guilabel:`Production Analysis` dashboard displays data using the *line chart* graph type of the
```

ksc-odoo replied:
might want to consider adding a graph-related icon here, as well

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+By default, the production analysis dashboard displays data using the *line chart* graph type of the
+*graph view*.
+
+To select the bar chart or pie chart graph type instead, click the :icon:`fa-bar-chart`
```
```suggestion
To select the bar chart or pie chart graph type, instead, click the :icon:`fa-bar-chart`
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+To select the bar chart or pie chart graph type instead, click the :icon:`fa-bar-chart`
+:guilabel:`(bar chart)`, or :icon:`fa-pie-chart` :guilabel:`(pie chart)` button below the
+:guilabel:`Search...` bar. To return to the line chart, click the :icon:`fa-line-chart`
+:guilabel:`line chart` button.
```
```suggestion
:guilabel:`(line chart)` button.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+:guilabel:`Search...` bar. To return to the line chart, click the :icon:`fa-line-chart`
+:guilabel:`line chart` button.
+
+To see production data displayed in a pivot table instead of a graph, click the
```
```suggestion
To see production data displayed in a pivot table, instead of a graph, click the
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+:guilabel:`line chart` button.
+
+To see production data displayed in a pivot table instead of a graph, click the
+:icon:`oi-view-pivot` :guilabel:`(pivot view)`, button to the right of the :guilabel:`Search...`
```
```suggestion
:icon:`oi-view-pivot` :guilabel:`(pivot view)` button, to the right of the :guilabel:`Search...`
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+To see production data displayed in a pivot table instead of a graph, click the
+:icon:`oi-view-pivot` :guilabel:`(pivot view)`, button to the right of the :guilabel:`Search...`
+bar. To return to graph view, click the :icon:`fa-area-chart` :guilabel:`(graph view)` button.
+
+Bar chart
+---------
```
might be cool to use anchors / ref links to these views, so the reader can quickly click-and-access them throughout the doc. Not required, just a suggestion to consider...

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+Bar chart
+---------
+
+The x-axis of the bar chart lists each week of the selected :guilabel:`End Date` time period. The
```
```suggestion
The x-axis of the bar chart lists each week of the selected time period. The
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+---------
+
+The x-axis of the bar chart lists each week of the selected :guilabel:`End Date` time period. The
+y-axis displays ascending numbers representing the count of whichever metric was selected on the
```
```suggestion
y-axis displays ascending numbers, representing the count of whichever metric was selected on the
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+
+The x-axis of the bar chart lists each week of the selected :guilabel:`End Date` time period. The
+y-axis displays ascending numbers representing the count of whichever metric was selected on the
+:guilabel:`Measure` drop-down menu (ex. Average Total Cost/Unit). Thus, each bar represents the
```
```suggestion
:guilabel:`Measures` drop-down menu (e.g. :guilabel:`Average Total Cost/Unit`). Thus, each bar represents the
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+:guilabel:`Measure` drop-down menu (ex. Average Total Cost/Unit). Thus, each bar represents the
+total count during a single week of the selected time period.
+
+Each bar is divided into sections representing one of the products manufactured during that week. By
```
```suggestion
Each bar is divided into sections, representing one of the products manufactured during that week. By
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+Each bar is divided into sections representing one of the products manufactured during that week. By
+default, the sections are stacked on top of one another to form a single bar.
+
+To view the sections as individual bars, click the :icon:`fa-database` :guilabel:`(stacked)` button
```
```suggestion
To view the sections as individual bars, click the :icon:`fa-database` :guilabel:`(stacked)` button,
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+
+To view the sections as individual bars, click the :icon:`fa-database` :guilabel:`(stacked)` button
+to the right of the graph view buttons, to deactivate the stacking feature. Click the button again
+to return to the stacked view.
```
I'm wondering if 'stacked view' should be capitalized...or hit with italics... 🤔 it might just be good as-is. But, it's something to think about during your revisions. 👍

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+
+Mouse over a bar, or a section of a bar, to see a popover providing a few details about it:
+
+- The selected :guilabel:`Measure`.
```
```suggestion
- The selected option from the :guilabel:`Measures` drop-down menu.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+- The selected :guilabel:`Measure`.
+- The week and year.
+- The product produced.
+- The total count of the selected :guilabel:`Measure`, during the week and year.
```
```suggestion
- The total count of the selected :guilabel:`Measures` option, during the week and year.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+Line chart
+----------
+
+The x-axis of the line chart lists each week of the selected :guilabel:`End Date` time period. The
```
```suggestion
The x-axis of the line chart lists each week of the selected time period. The
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+----------
+
+The x-axis of the line chart lists each week of the selected :guilabel:`End Date` time period. The
+y-axis displays ascending numbers representing the count of whichever metric was selected on the
+:guilabel:`Measure` drop-down menu (ex. Total Component Cost). Thus, each point on the chart
+represents the total count during a single week of the selected time period.
```
```suggestion
y-axis displays ascending numbers representing the count of whichever metric was selected from the
:guilabel:`Measures` drop-down menu (e.g. :guilabel:`Total Component Cost`). Thus, each point on the chart
represents the total count during a single week of the selected time period.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+represents the total count during a single week of the selected time period.
+
+A uniquely colored line appears on the chart for each product manufactured during the selected
+:guilabel:`End Date` time period, connecting the points for each week. By default, the points on the
```
```suggestion
time period, connecting the points for each week. By default, the points on the
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+
+To view the points unstacked, click the :icon:`fa-database` :guilabel:`(stacked)` button to the
+right of the graph view buttons, to deactivate the stacking feature. Click the button again to
+return to the stacked view.
```
same curious note re: 'Stacked' view as above ☝️

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+right of the graph view buttons, to deactivate the stacking feature. Click the button again to
+return to the stacked view.
+
+Mouse over a week to see a popover providing a few details about the products produced during it:
```
```suggestion
Mouse over a week to see a popover window providing a few details about the products produced during it:
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+
+Mouse over a week to see a popover providing a few details about the products produced during it:
+
+- The selected :guilabel:`Measure`.
```
```suggestion
- The selected option from the :guilabel:`Measures` drop-down menu.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+  - The color used to represent the product.
+  - The week and year.
+  - The name of the product.
+  - The total count of the selected :guilabel:`Measure`, during the week and year.
```
```suggestion
  - The total count of the selected :guilabel:`Measures` option, during the week and year.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+---------
+
+The pie chart displays one slice of pie for each product produced during each week of the selected
+:guilabel:`End Date` time period.
```
```suggestion
time period.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+:guilabel:`End Date` time period.
+
+.. example::
+   The month of February is selected in the :guilabel:`End Date` section of the
+   :guilabel:`Search...` bar. Units of a *chair* product and units of a *table* product were
+   manufactured during each of the four weeks of the month.
```
```suggestion
   The month of February is selected from the :guilabel:`End Date` drop-down menu, located in the :guilabel:`Filters` column of the
   :guilabel:`Search...` bar mega menu of filters and grouping options. Units of a *chair* product, and units of a *table* product, were
   manufactured during each of the four weeks of the month.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+   This means that the pie chart shows eight slices. Each slice represents one of the two products,
+   and one of the four weeks in which it was produced.
+
+Mouse over a pie slice to see a popover providing a few details:
```
```suggestion
Mouse over a pie slice to see a popover window providing a few details:
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+
+Mouse over a pie slice to see a popover providing a few details:
+
+- The selected :guilabel:`Measure`.
```
```suggestion
- The selected option from the :guilabel:`Measures` drop-down menu.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+- The selected :guilabel:`Measure`.
+- The week and year.
+- The product produced.
+- The total count of the selected :guilabel:`Measure`, during the week and year.
```
```suggestion
- The total count of the selected :guilabel:`Measures` option, during the week and year.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+- The week and year.
+- The product produced.
+- The total count of the selected :guilabel:`Measure`, during the week and year.
+- The percentage of the slice relative to the rest of the pie chart.
```
```suggestion
- The percentage of the slice, relative to the rest of the pie chart.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+- :guilabel:`Quantity Produced`
+
+To change the data categories displayed, click the :guilabel:`Measures` :icon:`fa-caret-down`
+button, and select or deselect categories from the popover window that appears.
```
```suggestion
button, and select (or deselect) categories from the popover window that appears.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+Click one of the column titles to sort the data from the lowest to the highest value. Click the
+title again to sort from the highest to the lowest value.
+
+To make the rows of the pivot table display time periods instead of products, begin by clicking the
```
```suggestion
To make the rows of the pivot table display time periods, instead of products, begin by clicking the
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+Use case: compare products
+==========================
+
+One of the best uses for the production analysis report is comparing statistics about two or more
```
```suggestion
One of the best uses for the :guilabel:`Production Analysis` report is comparing statistics about two or more
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+selecting the necessary measure, filter, and grouping, to see the desired data.
+
+.. example::
+   Toy manufacturer *Tommy's Toys* is trying to reduce their manufacturing operation costs. To
```
```suggestion
   Toy manufacturer, *Tommy's Toys*, is trying to reduce their manufacturing operation costs. To
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+
+.. example::
+   Toy manufacturer *Tommy's Toys* is trying to reduce their manufacturing operation costs. To
+   accomplish this, they've decided to identify redundant products and cease manufacturing the ones
```
```suggestion
   accomplish this, they have decided to identify redundant products, and cease manufacturing the ones
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+   with higher operation costs.
+
+   Two of the toys that have been singled out for analysis are the *pogo stick* and *moon shoes*.
+   Tommy's Toys believes that these two toys are so similar that they can stop manufacturing one,
```
```suggestion
   Tommy's Toys believes these two toys are so similar that they can stop manufacturing one,
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+   Tommy's Toys believes that these two toys are so similar that they can stop manufacturing one,
+   without significantly impacting their product offering.
+
+   To compare operation costs for the toys, business analyst Mike opens the
```
```suggestion
   To compare operation costs for the toys, business analyst, Mike, opens the
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+   without significantly impacting their product offering.
+
+   To compare operation costs for the toys, business analyst Mike opens the
+   :guilabel:`Manufacturing` app and navigates to the :guilabel:`Production Analysis` page. In the
```
```suggestion
   :menuselection:`Manufacturing app`, and navigates to the :guilabel:`Production Analysis` page. In the
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+
+   To compare operation costs for the toys, business analyst Mike opens the
+   :guilabel:`Manufacturing` app and navigates to the :guilabel:`Production Analysis` page. In the
+   :guilabel:`Search...` bar, he enters the names of both products. Then, he opens the drop-down
+   menu, and clicks :guilabel:`Product` in the :guilabel:`Group By` section.
```
```suggestion
   :guilabel:`Search...` bar, he enters the names of both products. Then, he opens the :guilabel:`Search...` bar drop-down
   menu of filter and grouping options, and clicks :guilabel:`Product` in the :guilabel:`Group By` section.
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+   drop-down menu, and selects the :guilabel:`Total Operation Cost/Unit` option. Finally, he selects
+   the :icon:`fa-bar-chart` :guilabel:`(bar chart)` graph type.
+
+   With these options selected, the production analysis report shows a bar chart for the current
```
```suggestion
   With these options selected, the :guilabel:`Production Analysis` report shows a bar chart for the current
```

ksc-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+   year, with one bar for each product, signifying the average operation cost for one unit of the
+   product.
+
+   With this data, Mike is able to see that the average operation cost for the moon shoes is almost
```
I know you've told me that certain formatting decisions have been made in the past with example admonitions like this that repeatedly mention a company or product, but I *do*, personally, think it looks better if, at the very least, the product name is capitalized... but, since you've said that you've been told to do otherwise, I guess this is just a pure suggestion to consider...

ksc-odoo reviewed (approved):
Hey @hojo-odoo -- just finished my Final Review of this PR. Great work! There are a decent amount of comments/feedback that require your attention. If you have any questions about anything, don't hesitate to reach out. But, once you take care of those necessary changes, and implement the required modifications, feel free to tag this for Tech Review. Thanks! 👍

hojo-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+for those products is shown.
+
+In addition to typing in the :guilabel:`Search...` bar to refine the data shown, clicking the
+:icon:`fa-caret-down` :guilabel:`(toggle)` button on the right side of the bar opens a drop-down
```
I like to call it the "toggle" button because it turns into an up arrow when you click on it.

hojo-odoo commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,320 @@
+Previous Period` and :guilabel:`End Date: Previous Year`.
+
+To view data using one of the two comparisons, begin by selecting a time period in the
+:guilabel:`End Date` field of the :guilabel:`Search...` bar. Then, make sure to select either the
```
I think it's fine to just say the search bar.

hojo-odoo commented:
Hey @samueljlieber,

This PR is ready for tech review.

Thanks,
John

samueljlieber reviewed (changes requested):
Hi @hojo-odoo! Thank you for your work on this reporting doc on the Manufacturing apps production analysis report. While the depth of content is appreciated, much of this doc is essentials content and can reference the [Odoo essentials Reporting doc](https://www.odoo.com/documentation/17.0/applications/essentials/reporting.html).

The **Search bar options** (**Filters**, **Group By**, and **Comparison**) are not unique to this report, and if left in this doc, will likely be repeated in other future reporting docs (**Manufacturing -> Reporting -> Work Orders** or **Overall Equipment Effectiveness**).

Same thing with the **Views** sections (**Bar chart**, **Line chart**, **Pie chart**, and **Pivot view**), there isnt a need to explain how these views work in this specific report when it is already covered in the essentials documentation. 

The **Measures** section is important to keep, since these are specific to this report, and providing a definition for each measure is a valuable reference for users when creating their production analysis report.

All that to say, a restructure of these _essentials_ sections could quickly convert them into use cases for the **Use case** section.

A couple ideas I have are:

- **Use case: compare production costs**: use the _Comparison_ option to see the difference in production costs for each quarter or year. Bonus– view this data in a pivot table.
- **Use case: product demand**: use the _Quantity Demanded_ measure to see the product demand in a  _Line chart_ view overtime, grouped by **End Date** and **Product**.

---

I know you put a lot of work into these sections that I am requesting to be removed from this doc, and since it is high quality content that can still be used, I suggest opening up a PR to improve the essentials Reporting doc with this content, especially to include the _Comparison_ feature. 

My request to remove this stems from reducing duplicate documentation by referencing existing documentation, as well as JUBs request to avoid writing _essentials_ content outside of the essentials documentation.

Please tag me for another review once you have had a chance to rework this PR and address my comments. Thank you @hojo-odoo 🙏

hojo-odoo commented:
Hey @samueljlieber,

Thank you for your review. I have pushed an update addressing your comments. 

In the new version, I've removed everything that falls under the essentials scope. I've also added an admonition block pointing readers toward the reporting essentials doc. I'm going to push an update to that doc to expand it and add more detail to what's already there.

I've also added an extra example to this doc, detailing how to use the comparison features of the report to compare two time periods.

Thanks,
John

hojo-odoo commented:
Hey @StraubCreative,

Sam did the first round of tech review on this PR and requested some changes, which I've implemented. Since he's busy this week, can you finish the tech review?

Thanks,
John

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,160 @@
+  the product.
+- :guilabel:`By-Products Total Cost`: the total value of all by-products created by manufacturing
+  the product.
+- :guilabel:`Component Cost/Unit`: the average cost of the components required to produce
+  one unit of the product.
```
early line break
```suggestion
- :guilabel:`Component Cost/Unit`: the average cost of the components required to produce one unit
  of the product.
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,160 @@
+  the product.
+- :guilabel:`Component Cost/Unit`: the average cost of the components required to produce
+  one unit of the product.
+- :guilabel:`Cost/Unit`: the average cost of producing one unit of the product,
+  including component, employee, operation, and subcontracting costs.
```
early line break
```suggestion
- :guilabel:`Cost/Unit`: the average cost of producing one unit of the product, including component,
  employee, operation, and subcontracting costs.
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,160 @@
+  product.
+- :guilabel:`Total Operation Cost`: the cumulative amount spent on operations required to produce
+  the product.
+- :guilabel:`Total Operation Cost/Unit`: the average cost of the operations required to produce
+  one unit of the product.
```
early line break
```suggestion
- :guilabel:`Total Operation Cost/Unit`: the average cost of the operations required to produce one
  unit of the product.
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,160 @@
+   shows data for the previous time period, quarter one of 2024.
+
+   .. note::
+      If :guilabel:`End Date: Previous Year` is selected instead of :guilabel:`End Date:
+      Previous Period`, the inner circle shows data for the selected time period, one *year*
+      previous.
```
early line break
```suggestion
      If :guilabel:`End Date: Previous Year` is selected instead of :guilabel:`End Date: Previous
      Period`, the inner circle shows data for the selected time period, one *year* previous.
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,160 @@
+the :guilabel:`Measures` :icon:`fa-caret-down` button, and selecting one of the options from the
+drop-down menu:
+
+The options displayed in the :guilabel:`Measures` drop-down menu, and the order they appear in,
```
missing down-caret here?
```suggestion
The options displayed in the :guilabel:`Measures` :icon:`fa-caret-down` drop-down menu, and the order they appear in,
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,160 @@
+  quantity demanded, represented as a percentage.
+- :guilabel:`Count`: the total count of |MOs| created for the product.
+
+.. note::
+   Only one measure can be selected at a time.
```
This is true only in Graph view. On the Pivot table view, we can select and cross-reference multiple Measures as well as Group-bys (which is the more likely use case since we can also export into a spreadsheet)
![image](https://github.com/user-attachments/assets/8ed52728-7cfb-4bc6-8051-5aa62a5beec7)

Suggestion to reframe as a tip, something like this (I didn't include tag formatting)
```suggestion
.. tip::
   Only one measure can be selected at a time on the Graph view, however, multiple measures and varying group-by criteria (across the x and y axis) can be selected for deeper analysis on the Pivot table view.
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,160 @@
+.. example::
+   Toy manufacturer *Tommy's Toys* is trying to reduce their manufacturing operation costs. To
+   accomplish this, they have decided to identify redundant products and cease manufacturing the
+   ones with higher operation costs.
```
nitpick, I think "operational" would be the descriptor for the subject "costs"
```suggestion
   ones with higher operational costs.
```

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,160 @@
+.. note::
+   Only one measure can be selected at a time.
+
+Use case: compare products
```
good example 👍

StraubCreative commented on content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst:
```diff
@@ -0,0 +1,160 @@
+      :align: center
+      :alt: The bar chart comparing the operation costs of the pogo stick and moon shoes.
+
+Use case: compare time periods
```
Cool, didn't know graph view could be used for this.

StraubCreative reviewed (approved):
Hi @hojo-odoo 

Looks good, I think your new additions hit the mark and are relevant for this dashboard.

I have a few items for you below before we can merge, some are necessary (line breaks mostly, there's a comment about the **note** too). Everything else is for your consideration.

As a follow-up I think there's an opportunity here to do detailed drill-down walkthroughs of specific reports using the Pivot table view since we can select multiple **Measures** and **Group By**s there. 

Here's an example of that in CRM app where there are multiple Filters, Measures, and Group Bys used to produce a rich Marketing Attribution report that really tells a lot, and is exportable to a spreadsheet:
![image](https://github.com/user-attachments/assets/0fa8dc18-3885-4c6a-8798-d53e15a9d1a4)

Can review this more if you like, otherwise feel free to merge after addressing the priority items below and anything else you agree with, thanks!

@robodoo delegate=hojo-odoo

samueljlieber reviewed (approved):
Hi @hojo-odoo approving after reading through the built documentation, and ZST did the technical review. Looks good to me! 👍 Thanks :)

hojo-odoo commented:
@robodoo r+
